core/src/test/java/org/apache/accumulo/core/iterators/DefaultIteratorEnvironment.java

```diff
@@ -18,17 +18,16 @@
 
 import java.io.IOException;
 
+import org.apache.accumulo.core.client.impl.TestIteratorEnvironment;
 import org.apache.accumulo.core.conf.AccumuloConfiguration;
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Value;
-import org.apache.accumulo.core.iterators.IteratorUtil.IteratorScope;
 import org.apache.accumulo.core.iterators.system.MapFileIterator;
-import org.apache.accumulo.core.security.Authorizations;
 import org.apache.accumulo.core.util.CachedConfiguration;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 
-public class DefaultIteratorEnvironment implements IteratorEnvironment {
+public class DefaultIteratorEnvironment extends TestIteratorEnvironment {
```

Comment:
Fixed in 3d0be31.  I think I ended with the goofy name because when I was re-factoring I initially created the class in src/test.   Then later I realized it was more generally useful and moved it.   Anyway, good catch.  I think the new name is much much better.